src/components/Image/index.tsx

```diff
@@ -0,0 +1,19 @@
+import {ImageSize, ImageStyled} from './index.styled';
+import Placeholder from '../../assets/images/add-avatar.svg';
+
+interface Props {
+  src?: string | null;
+  rounded?: boolean;
+  framed?: boolean;
+  size?: ImageSize
+}
+
+const Image = ({src, rounded = true, framed = true, size = 0 }: Props) => {
+  return (
+    <ImageStyled rounded={rounded} framed={framed} size={size}>
+        <img src={src || Placeholder} alt="Img" />
+    </ImageStyled>
+  )
+}
```

Comment:
General component for image is rounded and framed by default. Is it really ok?
Why do we need the div wrapper for image?
Also why do we need size 0 by default and at all?
How could I set some custom width and height? Maybe we will need some image with no 1:1 aspect ratio and so on

Reply:
Generally I agree with you, but well, there're some limitation it will "provide" you with in exchange of integrity and simplicity. 
- Currently in 3 of 3 usages we have rounded images, one of them is framed, there're defaults, but you can override those. If you prefer those flags to be set to false by default - ok, np, will do that :)
- div wrapper is the only (according to the quick web research) that will help me with round outer frame in Safari. Almost sure that might be done in different way, but I don't know how. 
-  zero size is just to have something in `default`, might replace it will `Small` or something. In fact' you shouldn't be able to get in this case, coz size has default value which is `Small` and it's an enum type, so you can't put random shit in there.
- Custom size is what I was thinking about and, well, we won't have it in this component. It will bring chaos only, all the icons would be of different size (that's not ok IMO), and controllable size is actually nice and flexible and so on. 

Now, last but not least: this isn't the only way to display images we mush use. It has limitation, although it will cover 90%+ of our cases, for the rest (non 1-1 ration and custom sizes) should be implemented separately. I still like this component despite it isn't perfect. It's definitely much better than 3 equivalent component done in 3 places in 3 different but similar ways.

Reply:
I think the best solution in this case is to rename this component. Let it be something like 'avatar', I don't know. And leave everything like now (except maybe default size). Those things bother me only if this component is suppose to be used like general image component across all project.

What do you think? For example, round outer frame on Safari is a good point, ok. But I don't think we will need it for the most of our images. So it is better to have such div wrapper like an exception, but not like general approach

Reply:
Ok, let’s try. I also thought it might be good as uploaded files miniatures (framed, but squared), which would be opened fully on click, but i guest we would figure something out for them

Reply:
Done